src/Domain/SalesforceWriteProxyRepository.php

```diff
@@ -107,7 +107,7 @@ public function push(SalesforceWriteProxy $proxy, bool $isNew): bool
      * @param int $limit    Maximum of each type of pending object to process
      * @return int  Number of objects pushed
      */
-    public function pushSalesforcePending(\DateTimeImmutable $now, int $limit = 200): int
+    public function pushSalesforcePending(\DateTimeImmutable $now, int $limit = 400): int
```

Comment:
Not a blocker, none of the callers to this actually pass $limit, so for simplicity it might be worth removing the param and making it a class constant, or even just an inline literal.

Reply:
e.g. something like this?

```php
     *
     * @return int  Number of objects pushed
     */
    public function pushSalesforcePending(\DateTimeImmutable $now): int
    {
        // We don't want to push donations that were created or modified in the last 5 minutes,
        // to avoid collisions with other pushes.
        $fiveMinutesAgo = $now->modify('-5 minutes');

        /** @var SalesforceWriteProxy[] $proxiesToCreate */
        $proxiesToCreate = $this->findBy(
            criteria: ['salesforcePushStatus' => SalesforceWriteProxy::PUSH_STATUS_PENDING_CREATE],
            orderBy: ['id' => 'ASC'],
            limit: 200,
        );
```

Reply:
@bdsl Makes sense. It's used twice so have gone with a class constant.